server/static/jsx/scholarship.jsx

```diff
@@ -0,0 +1,96 @@
+/** @jsx React.DOM */
```

Comment:
The JSX pragma shouldn't be necessary in React >= 0.12, I believe.

http://facebook.github.io/react/blog/2014/10/16/react-v0.12-rc1.html